### DISCUSSION TITLE
How to enable Dev UI when writing a Command Mode application

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [quarkusio/quarkus#42059] [Q&A]
I googled and looked around but did not find a way to enable Dev UI while writing a Command Mode application.

So is it even possible?
If yes, can we discuss here how to best do it, I'm happy to write something out of it for posterity.
Maybe a blog and a how to (or whatever these types of docs are called these days).

[a comment by an automated account is not shown]

**Reply by cescoffier** [reply]
It is not possible yet. We have a long-standing issue about it: https://github.com/quarkusio/quarkus/issues/33088
